cropper/src/main/java/com/canhub/cropper/BitmapUtils.kt

```diff
@@ -74,17 +74,12 @@ internal object BitmapUtils {
      * If no rotation is required the image will not be rotated.<br></br>
      * New bitmap is created and the old one is recycled.
      */
-    fun orientateBitmapByExif(bitmap: Bitmap?, context: Context, uri: Uri?): RotateBitmapResult {
-        var ei: ExifInterface? = null
-        try {
-            val `is` = context.contentResolver.openInputStream(uri!!)
-            if (`is` != null) {
-                ei = ExifInterface(`is`)
-                `is`.close()
-            }
-        } catch (ignored: Exception) {
+    fun orientateBitmapByExif(bitmap: Bitmap?, context: Context, uri: Uri): RotateBitmapResult {
+        val exifInterface = context.contentResolver.openInputStream(uri)?.use { ExifInterface(it) }
```

Comment:
Now we close every single time just like in https://github.com/CanHub/Android-Image-Cropper/pull/440